structure_tests/ext_run.sh

```diff
@@ -21,11 +21,13 @@ PULL=1
 CMD_STRING=""
 ENTRYPOINT="/test/structure_test"
 ST_IMAGE="gcr.io/gcp-runtimes/structure_test"
-USAGE_STRING="Usage: $0 [-i <image>] [-c <config>] [-v] [-e <entrypoint>] [--no-pull]"
+USAGE_STRING="Usage: $0 [-i <image>] [-c <config>] [-w <workspace>] [-v] [-e <entrypoint>] [--no-pull]"
 
 CONFIG_DIR=$(pwd)/.cfg
 mkdir -p "$CONFIG_DIR"
```

Comment:
technically yes, because the configs could still live outside of the workspace (they can be anywhere on the host filesystem). the workspace is also optional, i don't think we should require it